app/models/user.rb

```diff
@@ -46,8 +46,12 @@ def balances
       .reject { |bal| bal.balance == 0 && bal.pending_balance == 0 }
   end
 
-  def pending_orders_count
-    pending_user_orders_count + pending_user_dishes_count
+  def pending_orders_exist
```

Comment:
nit:
WDYTA `?` in the end of the method name?

Reply:
I know that by naming convention it should be with question mark, but considering that this method is passed to .jade's, its unable to handle `?` properly and i'm not sure what is possible workaround (and if it's worth of it)